pages/death.vue

```diff
@@ -0,0 +1,212 @@
+         descriptionShowing: false,
+         filters: {},
+         showing: false
+       },
+       hasWikiPage: {
+         label: "Wikipedia Page",
+         description: "Some of the accused witches have wikipedia pages written about them, to find out more about thier stories follow the link \"View Wiki Page\" when you are on an accused witch",
+         descriptionShowing: false,
+         filters: {
+           "hasWiki": {
+             "label": "Has wiki",
+             "active": true,
+             "iconUrl": "/images/witch-single-blue.png"
+           },
+           "noWiki": {
+             "label": "No wiki",
+             "active": true,
+             "iconUrl": "/images/witch-single-orange.png"
+           }
+         },
+         showing: false
+       }
+     }
+   }),
+   computed:{
+   icons() {
+      const { icons } = useIcons();
+      return icons.value;
+    }
+  },
+   methods: {
+     loadWikiEntries: function () {
+       const sparqlQuery = `SELECT DISTINCT ?item ?LabelEN ?page_title
+            WHERE {
+              ?item wdt:P4478 ?witch .
+              ?article schema:about ?item ; schema:isPartOf <https://en.wikipedia.org/> ;  schema:name ?page_title .
+              ?item rdfs:label ?LabelEN filter (lang(?LabelEN) = "en") .
+            }`;
+
+       const queryDispatcher = new SPARQLQueryDispatcher( this.sparqlUrl );
+       queryDispatcher.query( sparqlQuery ).then( result => {
+
+         for (let i = 0; i < result.results.bindings.length; i++) {
+           let item = result.results.bindings[i];
+
+           let wikiPage = {
+             id: item.item.value,
+             pageTitle: item.page_title.value,
+           }
+
+           this.wikiPages.push(wikiPage);
+         }
+
+       });
+    },
+    setMarkersIcons: function () {
+      let Filtering = new FilteringMethods(this.filterProperties, "sex");
+
+      for (let i = 0; i < this.originalMarkers.length; i++) {
+        let marker = this.originalMarkers[i];
+        [
+          marker.markerIcon,
+          marker.active
+        ] = Filtering.getMarkerStateIconDependant(marker);
+      }
+    },
+    hasLocalStorageExpired: function () {
+      let hours = 24; // Reset when storage is more than 24hours
+      let now = new Date().getTime();
+      let setupTime = localStorage.getItem('setupTime');
+
+      return setupTime === null || (now - setupTime > hours * 60 * 60 * 1000);
+    },
+    loadDataFromLocalStorage: function () {
+      let allFilters = JSON.parse(localStorage.getItem('allFilters'));
+      this.filterProperties.socialClass.filters = allFilters.socialClass;
+      this.filterProperties.occupation.filters = allFilters.occupation;
+    },
+    loadData: async function () {
+      this.loadWikiEntries();
+      const icons = this.icons;
+
+      try {
+        let response = await myFetch('/main.php?type=death')
+        console.log('API Response:', response);
```

Comment:
Think this might have been left in by mistake

Reply:
removed

Reply:
thank you